src/pages/workspaces/workspace/jobHistory/CallCacheWizard.js

```diff
@@ -0,0 +1,270 @@
+  const [metadataFetchError, setMetadataFetchError] = useState()
+  const [diffError, setDiffError] = useState()
+
+  const signal = Utils.useCancellation()
+
+  /*
+   * Data Fetchers
+   */
+
+  const readCalls = async otherWf => {
+    try {
+      const includeKey = [
+        'end', 'start', 'executionStatus'
+      ]
+      const excludeKey = []
+      const wf = await Ajax(signal).CromIAM.workflowMetadata(otherWf, includeKey, excludeKey)
+      setOtherWorkflowMetadata(wf)
+    } catch (error) {
+      console.log(error)
+      if (error instanceof Response) setMetadataFetchError(await error.text())
+      else if (_.isObject(error)) setMetadataFetchError(JSON.stringify(error))
+      else setMetadataFetchError(error)
+    }
+  }
+
+  const fetchDiff = async (otherWf, otherCall, otherIx) => {
+    try {
+      const diff = await Ajax(signal).CromIAM.callCacheDiff(workflowId, callFqn, Number(index), otherWf, otherCall, Number(otherIx))
+      setDiff(diff)
+    } catch (error) {
+      console.log(error)
+      if (error instanceof Response) setDiffError(await error.text())
+      else if (_.isObject(error)) setDiffError(JSON.stringify(error))
+      else setDiffError(error)
+    }
+  }
+
+  const otherCallFqnSelectionOptions = _.flow(
+    _.keys,
+    _.map(name => { return { value: name, label: name } })
+  )
+
+  const resetDiffResult = () => {
+    setDiff(undefined)
+    setDiffError(undefined)
+  }
+
+  const resetCallSelection = () => {
+    resetDiffResult()
+    setOtherCallFqn(undefined)
+    setOtherIndex(undefined)
+    setOtherAttemptSucceeded(undefined)
+    setOtherCallSelected(false)
+  }
+
+  const resetWorkflowSelection = () => {
+    resetCallSelection()
+    setOtherWorkflowId(undefined)
+    setMetadataFetchError(undefined)
+  }
+
+  /*
+   * Page render
+   */
+
+  const divider = hr({ style: { width: '100%', marginTop: '1rem', marginBottom: '1rem', border: '1px ridge lightgray' } })
+
+  const step1 = () => {
+    return h(Fragment, [
+      div({ style: { paddingTop: '0.5rem', fontSize: 16, fontWeight: 500, ...Style.noWrapEllipsis } }, ['Step 1: Select the workflow you expected to cache from']),
+      div({ style: { marginTop: '0.5rem', marginBottom: '1rem', display: 'flex', flexDirection: 'row', alignItems: 'center' } }, [
+        div({ style: { paddingRight: '0.5rem' } }, ['Workflow ID:']),
+        div({ style: { paddingRight: '0.5rem', flex: '2 1 auto' } }, [h(TextInput, { style: Style.codeFont, id: 'otherWorkflowId' })]),
+        div([h(ButtonPrimary, {
+          style: { float: 'right' },
+          onClick: () => {
+            const otherWf = document.getElementById('otherWorkflowId').value
+            setOtherWorkflowId(otherWf)
+            readCalls(otherWf)
+          }
+        }, ['Continue >'])])
+      ]),
+      div({ style: { paddingTop: '0.5rem', fontSize: 16, fontWeight: 500, color: 'lightgray', ...Style.noWrapEllipsis } }, ['Step 2: Select the call you expected to cache from']),
```

Comment:
I like that idea. Yes, the idea was "show which step you're currently in so that people can tell how far through the process they are... but since it's two-steps that probably shouldn't really be a concern and I like the tidiness of not showing it at all